Core/src/BareFatJets.cpp

```diff
@@ -3,24 +3,54 @@
 
 
 BareFatJets::BareFatJets(): BareP4(){
-    p4 = NULL;
-    rawPt = NULL;
-    flavour = NULL;
-    tau1 = NULL;
-    tau2 = NULL;
-    tau3 = NULL;
-    trimmedMass = NULL;
-    prunedMass = NULL;
-    filteredMass = NULL;
-    softdropMass = NULL;
-    ak8_subjet = NULL;
-    ak8jet_hasSubjet = NULL;
-    ak8subjet_btag = NULL;
-    hbb = NULL;
-
 }
 
```

Comment:
No, delete will do nothing if the pointer is properly zero initialized. The initialization is done in the .h file.